src/frontend/src/catalog/pg_catalog/pg_namespace.rs

```diff
@@ -23,5 +23,5 @@ pub const PG_NAMESPACE_TABLE_NAME: &str = "pg_namespace";
 pub const PG_NAMESPACE_COLUMNS: &[PgCatalogColumnsDef] = &[
     (DataType::Int32, "oid"),
     (DataType::Varchar, "nspname"),
-    (DataType::Varchar, "nspowner"), // TODO: using user id instead and support acl.
+    (DataType::Int32, "nspowner"),
```

Comment:
Are we supposed to have any tests on these 2 pg_catalog things? It feels fragile that no tests need to be updated in this PR given this change.